pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -180,3 +176,39 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 func multiply(x *inf.Dec, y *inf.Dec) *inf.Dec {
 	return inf.NewDec(0, 0).Mul(x, y)
 }
+
+// wrappedLimitRangerActions wraps the default LimitRangerActions.
+type wrappedLimitRangerActions struct {
+	internalLimitRangerActions limitranger.LimitRangerActions
+}
+
+// wrappedLimitRangerActions implements the LimitRangerActions interface
+var _ limitranger.LimitRangerActions = &wrappedLimitRangerActions{}
+
+// Limit is a pluggable function to enforce limits on the object.  This wraps the default
+// implementation to always return success so that all defaults will be applied.  Validation will
```

Comment:
It's not clear from usage what this does (the godoc doesn't give enough detail).  Why does this need to wrap the returns?